vms/administrator/utils.py

```diff
@@ -14,3 +16,23 @@ def wrapped_view(request, *args, **kwargs):
         return func(request, *args, **kwargs)
 
     return wrapped_view
+
+
+def admin_id_check(func):
+    @wraps(func)
+    def wrapped_view(request, admin_id):
+        administrator = getattr(request.user, 'administrator',
+                      hasattr(request.user, 'administrator'))
+        if administrator is None:
```

Comment:
what if administrator is set ?

Reply:
Can you please explain what you mean? @tonythomas01

Reply:
@tonythomas01 Please explain what you mean.